src/runtime_src/doc/toc/p2p.rst

```diff
@@ -151,3 +151,133 @@ PCIe Topology Considerations
 For best performance peer devices wanting to exchange data should be under the same PCIe switch.
 
 If IOMMU is enabled then all peer-to-peer transfers are routed through the root complex which will degrade performance significantly.
+
+
+P2P Data Transfer between FPGA Cards
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OpenCL coding style
+...................
+
+Consider the example situation as below:
+
+  - P2P data transfer from Card1 to Card2
+  - Source buffer (buf_src) is OpenCL buffer resident of Card1's DDR
+  - Destination buffer (buf_dst) is OpenCL buffer resident of Card2's DDR
+
+Typical coding style:
+
+  1. In the OpenCL host code, create separate `cl_context` for each `cl_device_id`
+  2. Define `buf_src` as regular buffer
+  3. Define `buf_dst` as P2P buffer
+  4. Import the P2P buffer or buf_dst to the context of buf_src. Use the following APIs
+
+       - `xclGetMemObjectFd`
+       - `xclGetMemObjectFromFd`
+  5. Perform the copy operation from `buf_src` to `imported_dst_buf`
+
+.. code-block:: cpp
+
+   cl_mem src_buf; // Source Buffer (regular) allocated in DDR bank0
+   cl_mem_ext_ptr_t src_buf_ext = {0};
+   src_buf_ext.flags = XCL_MEM_DDR_BANK0;
```

Comment:
since this is obsolete, maybe we should avoid using XCL_MEM_DDR_BANKXXX flags?

Reply:
OK, I removed this and also added clSetKernelArg which helps for identifying buffer location